src/Microsoft.Identity.Client/Instance/AdfsAuthority.cs

```diff
@@ -48,7 +48,7 @@ public AdfsAuthority(IServiceBundle serviceBundle, AuthorityInfo authorityInfo)
 
         internal override string GetTenantId()
         {
-            return null;
+            return String.Empty;
```

Comment:
As mentioned previously, a null string is semantically identical to an empty string. We should always check with String.IsNullOrEmpty 

So while this may fix the problem, the right fix would be to fix the logic that uses GetTenantId so that is robust, i.e. can handle nulls as well as empty strings.

Reply:
Also consider adding a bool SupportsMultipleTenants to the IAuthority abstraction if needed. This may make the code clearer.